web/src/pages/administration/programs/DocumentationsPage.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div class="d-flex mt-3">
+    <DocumentationsSearchableAutocomplete
+      v-model="documentationsIds"
+      :query="{
+        filters: {
+          notInProgram: programIdNumber,
+        },
+      }"
```

Comment:
You'll probably want to update this to a computed property to avoid weird re-render per-tick bugs.